### PR TITLE
[NET 6 Attribute Conversion] OpenGLES

### PR DESCRIPTION
**Pull request by tj-devel709**

### CODE REVIEW

File: src/OpenGLES/EAGLContext.cs [hunk 1]
```diff
@@ -22,7 +22,17 @@ public EAGLContext ()
 		}
 #endif
 
-#if !NET
```
Comment (tj-devel709) [inline src/OpenGLES/EAGLContext.cs:25]: Looks like `PresentRenderBuffer (nuint target, double presentationTime)` used to only be available for !NET, should this change be reverted?

Comment (mandel-macaque) [reply src/OpenGLES/EAGLContext.cs:25]: /cc @rolfbjarne

Comment (rolfbjarne) [reply src/OpenGLES/EAGLContext.cs:25]: What you did is correct!

**Review by rolfbjarne — APPROVED** [review]

**Review by chamons — APPROVED** [review]

**Comment by tj-devel709** [comment]
**Unrelated Test Failure**
- framework-test/Mac Catalyst [dotnet]/Debug [dotnet]: https://github.com/xamarin/maccore/issues/2414

**Pull request merged**